adaptations/Korg_Triton.py

```
@@ -0,0 +1,496 @@
+    data = _program_data_from_dump(message)
+    blanked = data[:]
+    blanked[:PROGRAM_NAME_LENGTH] = [0x20] * PROGRAM_NAME_LENGTH
+    if isSingleProgramDump(message):
+        return _build_single_program_dump(0, 0, 0, 0, blanked)
+    if isEditBufferDump(message):
+        return [0xF0, KORG, 0x30, TRITON_SERIES, CURRENT_PROGRAM_DUMP, _program_type(message)] + escapeSysex(blanked) + [0xF7]
+    raise Exception("Neither edit buffer nor program dump. Can't blank out.")
+
+
+def calculateFingerprint(message):
+    return hashlib.md5(bytearray(blankedOut(message))).hexdigest()
```

Ruff (0.15.10)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)